apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/source/StandardGrpcRdaSource.java

```diff
@@ -55,8 +55,7 @@ public class StandardGrpcRdaSource<TMessage, TClaim>
   /** The last sequence number range update timestamp. */
   private Instant lastSequenceNumberRangeUpdate = Instant.MIN;
 
-  /** The interval to update the sequence number range. */
-  private static final Long SEQUENCE_NUMBER_RANGE_UPDATE_MILLIS = 5000L;
+  private long sequenceRangeUpdateIntervalSeconds;
```

Comment:
While this is largely a self-documenting variable, is the removal of the comment intentional?